bears/r/FormatRBear.py

```diff
@@ -15,8 +16,38 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-
-        rcode = ("formatR::tidy_source(source =\""+filename+"\"")
+    def create_arguments(filename, file, config_file,
+                         r_comment: bool = True,
+                         r_blank: bool = True,
+                         r_brace: bool = False,
+                         r_arrow: bool = False,
+                         tab_width: int=SpacingHelper.DEFAULT_TAB_WIDTH,
+                         max_expr_length: int=20):
```

Comment:
this is definitely useful for other bears too, I'm not sure whether we have opened an issue now for collecting such names for inter-bear-use.